### PR TITLE
CSP-111 Moved the jobs project

### PR DESCRIPTION
**Pull request by DevenKShah**

### CODE REVIEW

File: src/SFA.DAS.Roatp.Api.UnitTests/Models/ProviderCourseEditModelTests.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using SFA.DAS.Roatp.Api.Models;
+using SFA.DAS.Roatp.Application.ProviderCourse;
+
+namespace SFA.DAS.Roatp.Api.UnitTests.Models
+{
+    [TestFixture]
+    public class ProviderCourseEditModelTests
+    {
+        [Test, AutoData]
+        public void ImplicitOperator_ReturnsCommant(ProviderCourseEditModel model)
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.Api.UnitTests/Models/ProviderCourseEditModelTests.cs:13]: typo 'Commant' should be 'Command'

Comment (MarkFCain) [inline src/SFA.DAS.Roatp.Api.UnitTests/Models/ProviderCourseEditModelTests.cs:13]: typo 'Commant' should be 'Command'

File: src/SFA.DAS.Roatp.Data/Repositories/ReloadStandardsRepository.cs [hunk 2]
```diff
@@ -38,13 +40,12 @@ public async Task<bool> ReloadStandards(List<Standard> standards)
                 await _roatpDataContext.BulkInsertAsync(standards);
                 await _roatpDataContext.SaveChangesAsync();
                 await transaction.CommitAsync();
-                _logger.LogInformation("Standards reload complete");
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.Data/Repositories/ReloadStandardsRepository.cs:41]: I think we should leave this log in, but I leave that decision to you

Comment (MarkFCain) [inline src/SFA.DAS.Roatp.Data/Repositories/ReloadStandardsRepository.cs:41]: I think we should leave this log in, but I leave that decision to you

**Review by MarkFCain — APPROVED** [review]
minor comments, nothing to hold up approval

File: src/SFA.DAS.Roatp.Data/Repositories/ReloadStandardsRepository.cs (hunk 2 above)
Comment (DevenKShah) [reply src/SFA.DAS.Roatp.Data/Repositories/ReloadStandardsRepository.cs:41]: I moved it further up in layers to confirm that all layers have seen the success

File: src/SFA.DAS.Roatp.Api.UnitTests/Models/ProviderCourseEditModelTests.cs (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.Roatp.Api.UnitTests/Models/ProviderCourseEditModelTests.cs]: Fixed

**Pull request merged**